process-engine/process_engine_runtime
# ✨ Add route for retrieving informtion about the host application

## Changes

Implement `getApplicationInfo` UseCase. Works for embedded ProcessEngineRuntimes, as well as standalone applications.

## Issues

Closes #531

PR: #532

## How to test the changes

- Call the service method
- See that it returns information about the ProcessEngineRuntime, even if it is embedded into another application
